### PR TITLE
Type annotations

### LINKED ISSUE #14: Enforce types

### PR DESCRIPTION
**Pull request by sulan**
Closes #14

### CODE REVIEW

**Comment by Error-42** [comment]
Shouldn't untyped defs be disallowed (disallow_untyped_defs)?

File: pyproject.toml [hunk 1]
```diff
@@ -27,3 +27,6 @@ disable = ["raw-checker-failed",
         "too-many-statements",
         "too-many-instance-attributes"]
 
+[tool.mypy]
+ignore_missing_imports = true
+check_untyped_defs = true
```
Comment (Error-42) [inline pyproject.toml:32]: Shouldn't untyped defs be disallowed (disallow_untyped_defs)?

**Review by Error-42 — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (sulan) [reply pyproject.toml:32]: We agreed not to enforce yet.

**Review by SzilBalazs — APPROVED** [review]
lgtm

**Review by SzilBalazs — APPROVED** [review]
lgtm

**Pull request merged**